.classpath

```diff
@@ -13,7 +13,7 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
```

Comment:
should be added to gitignore as it is redundant for intellij users
Why gitignore is added only on master?
Reviewer will checkout your PR and might be angry and pulling random stuff xD